cpp/src/parquet/encoding_test.cc

```diff
@@ -1094,16 +1094,26 @@ TYPED_TEST(TestByteStreamSplitEncoding, BasicRoundTrip) {
   // We need to test with different sizes to guarantee that the SIMD implementation
   // can handle both inputs with size divisible by 4/8 and sizes which would
   // require a scalar loop for the suffix.
-
-  // Exercise only the SIMD loop.
-  ASSERT_NO_FATAL_FAILURE(this->Execute(256, 1));
-
-  // Exercise both.
-  ASSERT_NO_FATAL_FAILURE(this->Execute(1337, 1));
-
-  for (int values = 0; values < 32; ++values) {
-    ASSERT_NO_FATAL_FAILURE(this->Execute(values, 1));
-  }
+  constexpr size_t kSuffixSize = 7;
```

Comment:
I'm not convinced all this is better than a loop over a range of possible sizes as above.